components/service-waiter/cmd/component.go

```diff
@@ -46,6 +48,11 @@ var componentCmd = &cobra.Command{
 		ctx, cancel := context.WithTimeout(cmd.Context(), timeout)
 		defer cancel()
 
+		if shouldSkipComponentWaiter(ctx) {
```

Comment:
Should not we call it each time inside the waitPodsImage loop?

Reply:
They behavior the same